TestServer/src/main/java/com/epam/TestServer.java

```diff
@@ -14,7 +11,7 @@
     private HttpServer httpServer;
 
     public void startServer() throws IOException {
-        httpServer = HttpServer.create(new InetSocketAddress("localhost",1234), 0);
+        httpServer = HttpServer.create(new InetSocketAddress("localhost",0), 0);
```

Comment:
formatting

Reply:
fixed in https://github.com/Mazel-Tovr/kotlin-app/pull/6